package.json

```diff
@@ -96,15 +87,22 @@
     "./js/util/web_worker.js": "./js/util/browser/web_worker.js"
   },
   "scripts": {
-    "build-dev": "browserify js/mapbox-gl.js --debug --ignore-transform unassertify --standalone mapboxgl > dist/mapbox-gl-dev.js && tap --no-coverage test/build/dev.test.js",
+    "build-dev": "browserify js/mapbox-gl.js --debug --standalone mapboxgl > dist/mapbox-gl-dev.js && tap --no-coverage test/build/dev.test.js",
+    "watch-dev": "watchify js/mapbox-gl.js --debug --standalone mapboxgl -o dist/mapbox-gl-dev.js -v",
+    "build-min": "browserify js/mapbox-gl.js --debug -t unassertify --plugin [minifyify --map mapbox-gl.js.map --output dist/mapbox-gl.js.map] --standalone mapboxgl > dist/mapbox-gl.js && tap --no-coverage test/build/min.test.js",
+    "watch-min": "watchify js/mapbox-gl.js --debug -t unassertify --plugin [minifyify --map mapbox-gl.js.map --output dist/mapbox-gl.js.map] --standalone mapboxgl -o dist/mapbox-gl.js -v",
+    "build-token": "browserify debug/access-token-src.js --debug -t envify > debug/access-token.js",
+    "watch-bench": "watchify bench/index.js --plugin [minifyify --no-map] -t [babelify --presets react] -t unassertify -t envify -o bench/bench.js -v",
+    "download-bench-data": "node bench/download-data.js",
+    "start-server": "st --no-cache --localhost --port 9966 --index index.html .",
```

Comment:
@mourner is there an easy way to increment the port until an open one is found?  Asking because `budo` by default uses the same one, and I've definitely run into collisions while developing on gl-js.